src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/filters/suppresswithplaintextcommentfilter/Example9.java

```diff
@@ -0,0 +1,30 @@
+/*xml
+<module name="Checker">
+  <module name="com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck">
```

Comment:
to hide this defect of our test suite, better to do 
https://github.com/checkstyle/checkstyle/pull/14462#discussion_r1493354985 , as eventually it will be resolved.

Reply:
Done